docs/workflows.md

```diff
@@ -405,14 +405,17 @@ The main reason for increasing the default value would be to accommodate a Workf
 ## Signals
 
 A Signal is an asynchronous request to a [Workflow Execution](#workflow-executions).
+Signals can be defined in code or executed on the command line.
 
-A Signal delivers data to a running Workflow Execution.
-It cannot return data to the caller; to do so, use a [Query](#queries) instead.
-The Workflow code that handles a Signal can mutate Workflow state.
-A Signal can be sent from a Temporal Client or a Workflow.
-When a Signal is sent, it is received by the Cluster and recorded as an Event to the Workflow Execution [Event History](#event-history).
-A successful response from the Cluster means that the Signal has been persisted and will be delivered at least once to the Workflow Execution.[^1]
-The next scheduled Workflow Task will contain the Signal Event.
+- [How to develop, send, and handle Signals in code](/application-development/features#signals)
```

Comment:
This seems like going back in time, to an older version of the text?

Reply:
Hmm ya, the links should now be there, but your content updates should be too.  Will fix this.